app/src/main/java/com/framgia/forder/data/model/OrderDetail.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.forder.data.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import static com.framgia.forder.utils.Constant.FORMAT_PRICE;
+import static com.framgia.forder.utils.Constant.UNIT_MONEY;
+
+public class OrderDetail {
+    @Expose
+    @SerializedName("id")
+    private int mId;
+    @Expose
+    @SerializedName("product")
+    private Product mProduct;
+    @Expose
+    @SerializedName("quantity")
+    private int mQuantity;
+    @Expose
+    @SerializedName("status")
+    private int mStatus;
+    private boolean mCheckBoxStatus;
+
+    public int getId() {
+        return mId;
+    }
+
+    public void setId(int id) {
+        mId = id;
+    }
+
+    public int getQuantity() {
+        return mQuantity;
+    }
+
+    public void setQuantity(int quantity) {
+        mQuantity = quantity;
+    }
+
+    public Product getProduct() {
+        return mProduct;
+    }
+
+    public void setProduct(Product product) {
+        mProduct = product;
+    }
+
+    public int getStatus() {
+        return mStatus;
+    }
+
+    public void setStatus(int status) {
+        mStatus = status;
+    }
+
+    public boolean isCheckBoxStatus() {
+        return mCheckBoxStatus;
+    }
+
+    public void setCheckBoxStatus(boolean checkBoxStatus) {
+        mCheckBoxStatus = checkBoxStatus;
+    }
+
+    public String getTotalPriceFormat() {
+        return String.format(FORMAT_PRICE, mProduct.getPrice() * mQuantity) + UNIT_MONEY;
```

Comment:
WARNING: [DefaultLocale] Implicitly using the default locale is a common source of bugs: Use `String.format(Locale, ...)` instead
 Calling `String#toLowerCase()` or `#toUpperCase()` *without specifying an explicit locale* is a common source of bugs. The reason for that is that those methods will use the current locale on the user's device, and even though the code appears to work correctly when you are developing the app, it will fail in some locales. For example, in the Turkish locale, the uppercase replacement for `i` is *not* `I`.

If you want the methods to just perform ASCII replacement, for example to convert an enum name, call `String#toUpperCase(Locale.US)` instead. If you really want to use the current locale, call `String#toUpperCase(Locale.getDefault())` instead.